src/main/java/culturemedia/controller/CultureMediaController.java

```diff
@@ -0,0 +1,28 @@
+package culturoteca.controllers;
+
+package culturemedia.controller;
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
+import culturoteca.model.Video;
+import culturoteca.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+    private final CultureMediaServiceImpl cultureMediaService;
+
+
+    public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+        this.cultureMediaService = cultureMediaService;
+    }
+
+
+    public List<Video> find_allVideos() throws VideoNotFoundException {
+        List<Video> videos = null;
+        videos = cultureMediaService.findAll();
```

Comment:
delate this line

Reply:
replase: if (videos.isEmpty()) {
            throw new VideoNotFoundException("No videos found.");
        }